js/CSVGood.js

```diff
@@ -196,6 +196,9 @@ const CSVGood = function (file, onStep, onError, onComplete) {
 
         fileReader.onload = (evt) => {
             // Take result
+            if (!evt.target) {
+                throw new Error("Expected event to have a target")
+            }
```

Comment:
I added all these types of checks to satisfy the type-checker errors thrown by the TS compiler where technically APIs can return `null`/`undefined`.